.vscode/tasks.json

```diff
@@ -6,7 +6,7 @@
         {
             "label": "build",
```

Comment:
Does it make sense to change the label `build` to `build with debug` to reflect the debug option?

Reply:
Added a build with coverage tasks and a description to both